htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -77,3 +77,6 @@ ALTER TABLE llx_recruitment_recruitmentcandidature ADD email_date datetime after
 ALTER TABLE llx_ticket ADD email_date datetime after email_msgid;
 
 INSERT INTO llx_const (name, entity, value, type, visible) VALUES ('MAIN_SECURITY_MAX_IMG_IN_HTML_CONTENT', 1, 1000, 'int', 0);
+
+ALTER TABLE llx_element_element CHANGE sourcetype sourcetype VARCHAR(64) NOT NULL;
```

Comment:
To change the type of a field, use syntax
ALTER TABLE llx_table MODIFY COLUMN name varchar(60);
(see header of migration file).
Use for CHANGE is to rename a column.

this rule is for better compatiblity and impact analysis searches.